src/components/flat-table/flat-table.pw.tsx

```diff
@@ -117,6 +117,11 @@ const tableSorting: ["first" | "second", string, number, number][] = [
   ["second", "asc", 2, 1],
 ];
 
+const basicColorThemes: [FlatTableProps["colorTheme"], string, string][] = [
```

Comment:
suggestion (non-blocking): looks like you could potentially `slice` this sub array off the `colorThemes` or you could do something like:

```
const basicColorThemes: [FlatTableProps["colorTheme"], string, string][] = [
  ["dark", "rgb(51, 91, 112)", "rgb(102, 132, 148)"],
  ["light", "rgb(204, 214, 219)", "rgb(179, 194, 201)"],
];

const colorThemes: [FlatTableProps["colorTheme"], string, string][] = [
  ...basicColorThemes,
["transparent-base", "rgb(242, 245, 246)", "rgb(242, 245, 246)"],
  ["transparent-white", "rgb(255, 255, 255)", "rgb(255, 255, 255)"],
  ];